### PR TITLE
[ID] Update Users with unique titles

### PR DESCRIPTION
**Pull request by hikikuman**
Changed MWC 2019 to MWC 2020 also the list. Added hypercyte to Official mapping contest winners list. Added fanzhen0019 and Mr HeliX to Aspire mapping contest winners list. Linking nekodex osu! profile at Featured Artists. Added Namirin to Featured Artists list. Added xiemon to Mascot design contest winners list. Updated "By name" list from en.md. All list have been compared with en.md.

### CODE REVIEW

File: wiki/People/Users_with_unique_titles/id.md [hunk 1]
```diff
@@ -290,35 +297,34 @@ Para pemain ini mendapatkan gelar mereka melalui pencapaian tergila dalam game i
...
 | ![][flag_SE] [Xgor](https://osu.ppy.sh/users/98661) | Elite Mapper |
-| ![][flag_KR] [Yeoul](https://osu.ppy.sh/users/4698728) | osu!mania champion |
+| ![][flag_ID] [xiemon](https://osu.ppy.sh/users/5203667) | osu!artist |
 | ![][flag_HM] [Zallius](https://osu.ppy.sh/users/55) | Boats |
 | ![][flag_DE] [Zetera](https://osu.ppy.sh/users/587737) | Medal Hunter |
 | ![][flag_CL] [ZiRoX](https://osu.ppy.sh/users/200768) | Elite Mapper |
 
 [flag_AT]: /wiki/shared/flag/AT.gif "Austria"
 [flag_AU]: /wiki/shared/flag/AU.gif "Australia"
+[flag_BR]: /wiki/shared/flag/BR.gif "Brazil"
```
Comment (fajar13k) [inline wiki/People/Users_with_unique_titles/id.md:327]:
```suggestion
[flag_BR]: /wiki/shared/flag/BR.gif "Brasil"
```

**Review by fajar13k — APPROVED** [review]
Itu aja kayaknya

File: wiki/People/Users_with_unique_titles/id.md (hunk 1 above)
Comment (hikikuman) [reply wiki/People/Users_with_unique_titles/id.md:327]: iya soalnya sisanya sama, kebanyakan cuma nambahin nama dan ngubah mwc 2019 jadi mwc 2020

**Review by fajar13k — APPROVED** [review]

**Review by cl8n — APPROVED** [review]

**Comment by cl8n** [comment]
fyi the BR, CN, and RU flags have different text here than in meta/flag-references, should decide what's more correct and update that accordingly

**Pull request merged**